website/src/widgets/quickstart-training.js

```diff
@@ -9,8 +9,8 @@ import { htmlToReact } from '../components/util'
 const DEFAULT_LANG = 'en'
 const DEFAULT_HARDWARE = 'cpu'
 const DEFAULT_OPT = 'efficiency'
-const DEFAULT_TEXTCAT_EXCLUSIVE = true
-const COMPONENTS = ['tagger', 'parser', 'ner', 'textcat']
+const DEFAULT_TEXTCAT_EXCLUSIVE = false
```

Comment:
After Ines her fix to `master`, this edit can be reverted:
```suggestion
const DEFAULT_TEXTCAT_EXCLUSIVE = true
```